feedback.html

```diff
@@ -0,0 +1,181 @@
+                            <div class="col-md-6">
+                                <div class="btn-group w-100" role="group" aria-label="Did you enjoy your experience?">
+                                    <input type="radio" id="experience-yes" name="experience" class="toggle">
+                                    <label for="experience-yes" class="btn btn-outline-primary toggle-yes">Yes</label>
+                                    <input type="radio" id="experience-no" name="experience" class="toggle">
+                                    <label for="experience-no" class="btn btn-outline-primary toggle-no">No</label>
+                                </div>
+                                <div class="invalid-feedback">Please select Yes/No</div>
+                            </div>
+                        </div>
+                            
+                        <div class="row mb-4">
+                            <div class="col-md-6">Would you play again?</div>
+                            <div class="col-md-6">
+                                <div class="btn-group w-100" role="group" aria-label="Would you play again?">
+                                    <input type="radio" id="playAgain-yes" name="playAgain" class="toggle">
+                                    <label for="playAgain-yes" class="btn btn-outline-primary toggle-yes">Yes</label>
+                                    <input type="radio" id="playAgain-no" name="playAgain" class="toggle">
+                                    <label for="playAgain-no" class="btn btn-outline-primary toggle-no">No</label>
+                                </div>
+                                <div class="invalid-feedback">Please select Yes/No</div>
+                            </div>
+                        </div>
+                
+                        <div class="row mb-4">
+                            <div class="col-md-6">Would you recommend to others?</div>
+                            <div class="col-md-6">
+                                <div class="btn-group w-100"  role="group" aria-label="Would you recommend to others?">
+                                    <input type="radio" id="recommend-yes" name="recommend" class="toggle">
+                                    <label for="recommend-yes" class="btn btn-outline-primary toggle-yes">Yes</label>
+                                    <input type="radio" id="recommend-no" name="recommend" class="toggle">
+                                    <label for="recommend-no" class="btn btn-outline-primary toggle-no">No</label>
+                                </div>
+                                <div class="invalid-feedback">Please select Yes/No</div>
+                            </div>
+                        </div>
+
+                        <div class="row mb-4">
+                            <div class="col-md-12">
+                                <label for="organizers">Who were the organizers?</label>
```

Comment:
The question wasn't who were the organizers xD
It is supposed to be: How supportive were the organizers?
Options:
Very supportive
Just right
Not at all